packages/vats/src/core/basic-behaviors.js

```diff
@@ -22,7 +22,7 @@ const RESERVE_ADDRESS = 'agoric1ae0lmtzlgrcnla9xjkpaarq5d5dfez63h3nucl';
  * non-exhaustive list of powerFlags
  * REMOTE_WALLET is currently a default.
  */
-const PowerFlags = /** @type {const} */ ({
+export const PowerFlags = /** @type {const} */ ({
```

Comment:
this is a funny name in this context. Worth a comment explaining the term comes from the Golang code. https://github.com/Agoric/agoric-sdk/blob/eb5b1f5b0635680081d708e1928c8ee089b61c80/packages/cosmic-proto/gen/agoric/swingset/msgs.ts#L50 ?